deploy-board/deploy_board/templates/environs/env_landing.html

```diff
@@ -231,7 +231,7 @@ <h4 class="modal-title">Resume Confirm</h4>
             },
             computed: {
                 capacityText: function() {
-                    if (this.remainingCapacity < 100) {
+                    if (this.remainingCapacity < 100 && this.remainingCapacity != null) {
```

Comment:
`this.remainingCapacity < 100 && this.remainingCapacity != null` works in JavaScript but it's a good practice to do the null check first.

Reply:
updated